otter/integration/lib/autoscale.py

```diff
@@ -100,11 +101,26 @@ def create_scaling_group_dict(
     Attribute('group_config', instance_of=dict),
     Attribute('pool', default_value=None),
     Attribute('reactor', default_value=None),
+    Attribute('treq', default_value=treq)
 ])
 class ScalingGroup(object):
     """This class encapsulates a scaling group resource.  It provides a means
     which lets you create new scaling groups and, later, automatically
     dispose of them upon integration test completion.
+
+    :ivar group_config: The complete JSON dictionary the group was
+        created with - a dictionary including 'groupConfiguration',
+        'launhConfiguration', and maybe 'scalingPolicies'
+
+    :ivar pool: a :class:`twisted.web.client.HTTPConnectionPool` to pass to
+        all treq requests
+
+    :ivar reactor: a :class:`twisted.internet.interfaces.IReactorTime`
+        provider, to be used for timeouts and retries
+
+    :ivar _treq: the treq module to use for making requests - if not provided,
```

Comment:
Here is the sphinx documentation for the Python domain - we usually leave out the :py: stuff since we assume it's understood:  http://sphinx-doc.org/domains.html#the-python-domain.

https://docs.python.org/devguide/documenting.html is also a good guide (it's the documentation standard for CPython I think).